### ISSUE TITLE
Elements of complex initializers (such as dictionary initializers) should be indented to the right side of the opening brace

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#1233]
We should file a warning in case an element within a complex initializer (such as for a dictionary) is not properly indented.

Example:
Following code should generate a warning:
```c#
var dictionary = new Dictionary<string, string>()
                     {
                         {
"some value", "some other value" },
                     };
```
It should be fixed into:
```c#
var dictionary = new Dictionary<string, string>()
                     {
                         {
                           "some value", "some other value" },
                     };
```

Note: It may be that StyleCop reports elements that span multiple lines to be on different lines than the opening brace.
So it should NOT be fixed into
```c#
var dictionary = new Dictionary<string, string>()
                     {
                         { "some value", "some other value" }
                     };
```
Instead, it shall only be indented to the right!
